jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -96,7 +96,9 @@ public void buildDocker()
             jibExtension.getTo().getImage(),
             eventDispatcher,
             getProject().getName(),
-            getProject().getVersion().toString(),
+            getProject().getVersion().toString().equals("unspecified")
+                ? "latest"
+                : getProject().getVersion().toString(),
```

Comment:
This works, but the argument in the Javadoc is
```
   * @param projectVersion the project version, as determined by the plugin
```
so "latest" as a `projectVersion` might not fit well (especially the Javadoc says "as determined by the plugin"). I wonder if it makes sense to either push the logic down to `getGeneratedTargetDockerTag()` or change the argument name to ... something else.

Reply:
If the version is unspecified in Maven, you get `[ERROR] Some problems were encountered while processing the POMs: [FATAL] 'version' is missing. @ line 4, column 104`, so I decided to just make the change gradle-side.

I will change the argument name.